packages/verkle/src/node/internalNode.ts

```diff
@@ -1,23 +1,23 @@
-import { equalsBytes } from '@ethereumjs/util'
+import { BIGINT_0, bytesToBigInt, equalsBytes } from '@ethereumjs/util'
 
 import { POINT_IDENTITY } from '../util/crypto.js'
 
 import { BaseVerkleNode } from './baseVerkleNode.js'
 import { LeafNode } from './leafNode.js'
 import { NODE_WIDTH, VerkleNodeType } from './types.js'
 
-import type { VerkleNode, VerkleNodeOptions } from './types.js'
+import type { VerkleCrypto } from '../types.js'
+import type { VerkleNodeOptions } from './types.js'
 
 export class InternalNode extends BaseVerkleNode<VerkleNodeType.Internal> {
-  // Array of references to children nodes
-  public children: Array<VerkleNode | null>
+  // Array of commitments to child nodes
+  public children: Uint8Array[]
   public copyOnWrite: Record<string, Uint8Array>
   public type = VerkleNodeType.Internal
 
-  /* TODO: options.children is not actually used here */
   constructor(options: VerkleNodeOptions[VerkleNodeType.Internal]) {
     super(options)
-    this.children = options.children ?? new Array(NODE_WIDTH).fill(null)
+    this.children = options.children ?? new Array(NODE_WIDTH).fill(new Uint8Array())
```

Comment:
We could use the `zeroValues` or `zeroCValues` constant that has been declared in one of the other files, right?